appIcons.js

```diff
@@ -255,8 +269,14 @@ export const TaskbarAppIcon = GObject.registerClass({
             SETTINGS.connect('changed::group-apps-label-font-color-minimized', this._updateWindowTitleStyle.bind(this)),
             SETTINGS.connect('changed::group-apps-label-max-width', this._updateWindowTitleStyle.bind(this)),
             SETTINGS.connect('changed::group-apps-use-fixed-width', this._updateWindowTitleStyle.bind(this)),
-            SETTINGS.connect('changed::group-apps-underline-unfocused', this._settingsChangeRefresh.bind(this))
-        ]
+            SETTINGS.connect('changed::group-apps-underline-unfocused', this._settingsChangeRefresh.bind(this)),
+        ];
+
+        if (!this._checkGtkVersion_cssVariables()) {
+            this._dtpSettingsSignalIds += [
```

Comment:
You are converting array into string

Reply:
I don't see your point here. `this._dtpSettingsSignalIds` is an array of integers (signal IDs to be precise) and I concatenate two arrays. It may be not the most elegant solution, but I can't see any array-to-string conversion (even implicitly) here.
Please advise